core/src/commonMain/kotlin/com/ustadmobile/core/util/ext/ClazzWithSchoolExt.kt

```diff
@@ -2,4 +2,16 @@ package com.ustadmobile.core.util.ext
 
 import com.ustadmobile.lib.db.entities.ClazzWithSchool
 
-fun ClazzWithSchool.effectiveTimeZone() = clazzTimeZone ?: school?.schoolTimeZone ?: "UTC"
+fun ClazzWithSchool.findClazzTimeZone(): String{
```

Comment:
What is the difference between the objective of this and effectiveTimezone ? 

Also I think there is a logic error here:
if school != null and the timezone is null or blank, it would give UTC instead of looking at the class time zone.